### DISCUSSION TITLE
Drag and drop to given position

### DISCUSSION THREAD

**Discussion by monkcoders** [Atri-Labs/atrilabs-engine#638] [Discuss: New Feature Ideas]
Currently, the atri framework allows users to only drag and drop the elements(like text, buttons, images etc.) to the page. Still, the positioning needs to be done with the help of CSS properties, which is difficult for the non-coder. We should develop a feature by which we can place the element directly to the position where we want it to be on the live web page and also allow the developer to do the current settings in the form of advanced settings. WordPress and canva can be considered as an example here. I also have some ideas by which we can perform this.

**Comment by cruxcode** [comment]
@monkcoders There is a strong rationale for why we don't want to do this. Our very first version used to have this function but it leads to bad CSS, hence, a bad website. The reason is that we will have to either use `position: absolute` or `transform: translate(x, y)` to achieve this drop-anywhere-in-canvas functionality. This will make the website unresponsive.
